nvtabular/ops/categorify.py

```diff
@@ -378,10 +381,10 @@ def set_storage_path(self, new_path, copy=False):
         self.out_path = new_path
 
     @annotate("Categorify_transform", color="darkgreen", domain="nvt_python")
-    def transform(self, columns: ColumnNames, df: DataFrameType) -> DataFrameType:
+    def transform(self, columns: Columns, df: DataFrameType) -> DataFrameType:
         new_df = df.copy(deep=False)
         if isinstance(self.freq_threshold, dict):
-            assert all(x in self.freq_threshold for x in columns)
+            assert all(x in self.freq_threshold for x in columns.names())
```

Comment:
Do we need to flatten this here? What happens if you have a tuple in this... will it check inside the tuple? is it supposed to?

Reply:
I was wondering the same! There are some other parts of the codebase where this might be an issue as well.